### PR TITLE
Seedlet: support filtering archive page titles

### LINKED ISSUE #2317: Seedlet: give users the ability to partially hide archive page headings
As of WordPress 5.5, theme authors can [filter of archive page headings](https://make.wordpress.org/core/2020/07/30/filtering-archive-pages-headings-in-wordpress-5-5/).

We should filter the header output in Seedlet so that we can give users an easy way to hide the prefix via CSS, if so desired.

Screen Shot 2020-07-30 at 3 26 07 PM

cc @kathrynwp

### PR DESCRIPTION
**Pull request by jffng**
This PR addresses #2317.

**To test**
- Activate seedlet and visit any page that uses the archive template
- Verify that the prefix is wrapped in a span with the CSS class `archive-prefix`

### CODE REVIEW

**Comment by scruffian** [comment]
The `$prefix` and `$original_title` params were only added in 5.5.0, so we'll need to change the minimum version supported if we make this change.

**Review by scruffian — APPROVED** [review]
LGTM

**Comment by jffng** [comment]
> we'll need to change the minimum version supported if we make this change.

Ah that's a good point. @jeffikus do you think this is acceptable to support #2317?

**Comment by jeffikus** [comment]
Being a future focused theme, I think it's reasonable to start support for it at 5.5.0.

**Comment by pbking** [comment]
Has a conflict and needs to updated minimum WP version to 5.5

File: seedlet/functions.php [hunk 1]
```diff
@@ -106,20 +106,11 @@ function seedlet_setup() {
 		// Add support for editor styles.
 		add_theme_support( 'editor-styles' );
 
-		$editor_stylesheet_path = './assets/css/style-editor.css';
-
-		// Note, the is_IE global variable is defined by WordPress and is used
-		// to detect if the current browser is internet explorer.
-		global $is_IE;
-		if ( $is_IE ) {
-			$editor_stylesheet_path = './assets/css/ie-editor.css';
```
Comment (jffng) [inline seedlet/functions.php:115]: Not sure why we are still checking this here because we don't use or have IE specific stylesheets anymore.

**Comment by jffng** [comment]
I brought this PR up-to-date and bumped the minimum required WP version of Seedlet to 5.5.0.

**Review by MaggieCabrera — APPROVED** [review]
This looks good to go

**Comment by pbking** [comment]
reviewed and rebased against trunk

**Pull request merged**